wampy/transports/websocket/connection.py

```diff
@@ -84,6 +87,12 @@ def receive(self, bufsize=1):
 
             try:
                 frame = ServerFrame(received_bytes)
+
+                if frame.opcode == 9:
```

Comment:
i think this would be more appropriate in the `else` clause. i've learnt something here -  The Ping frame contains an opcode of 0x9 - so thanks. If you can move this into the `else` clause and add a comment that explains what we are checking for that would be great.